backend/README.md

```diff
@@ -0,0 +1,15 @@
+# DroneCore with GRPC
+
+### Build with backend
+
+```
+make BUILD_BACKEND=YES
+```
+
+### Run the backend
+
+The backend is built as an executable:
+
+```
+./build/default/backend/src/backend_bin
```

Comment:
Are you sure you're placing backend executable in `src` ?

Reply:
I am not sure the reason behind having it in `src`. As you know, for DroneCore unit tests, we've in  `build/default/unit_tests_runner`.  I think `./build/default/backend/bin` is also fine.

Reply:
Right. I can put them in `build/default`, of course.

Reply:
Actually, I am not sure what a good practice would be for that (and also maybe `install` should be set for the backend). I don't have much time for this right now, I just wanted to move `grpc/server` to `backend` now that not so many people are working on it.

Maybe changing the build output/install targets can be done later?

Reply:
Yeah its perfectly fine.